x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecuritySystemIndices.java

```diff
@@ -821,8 +823,15 @@ private static Settings getProfileIndexSettings() {
             .put("analysis.filter.email.preserve_original", true)
             .putList("analysis.filter.email.patterns", List.of("([^@]+)", "(\\p{L}+)", "(\\d+)", "@(.+)"))
             .put("analysis.analyzer.email.tokenizer", "uax_url_email")
-            .putList("analysis.analyzer.email.filter", List.of("email", "lowercase", "unique"))
-            .build();
+            .putList("analysis.analyzer.email.filter", List.of("email", "lowercase", "unique"));
+        if (DiscoveryNode.isStateless(settings)) {
```

Comment:
Do we have integration tests which are running with `stateless.enabled` set to `true`?

Reply:
I assume so, but not in this repo.